client/stylesheets/_responsive.scss

```diff
@@ -403,6 +450,29 @@
     .profileCardLi {
         margin: 6px 5px !important;
     }
+
+    .profileChatBtn a, .optionsProClick ul .clicker {
+        font-size: 0.9rem !important;
+        padding: 12px 16px !important;
+    }
+
+    .box-title {
+        font-size: 0.9rem !important;
+    }
+
+    .optionsProClick {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `optionsProClick` should be written in lowercase with hyphens